exporter/sumologicexporter/exporter.go

```diff
@@ -374,6 +397,26 @@ func (se *sumologicexporter) handleUnauthorizedErrors(ctx context.Context, errs
 	}
 }
 
+func (se *sumologicexporter) pushTracesData(ctx context.Context, td ptrace.Traces) error {
+	logsURL, metricsURL, tracesURL := se.getDataURLs()
+	sdr := newSender(
```

Comment:
I'm not familiar with Sumo but why does it need to create a new sender for each trace batch, why not create a sender once and reuse it?

Reply:
We used to have implementation which would required synchronisation before, but now I think we could reuse it 🤔

I created issue for it: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33105